lib/generators/inertia/install/js_package_manager.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Inertia
+  module Generators
+    class JSPackageManager
+      def self.package_managers
+        %w[npm yarn bun pnpm]
```

Comment:
Gonna point to this file next time I need an example of why the JS ecosystem feels like the wild west compared to Ruby 😅